istio/addons.go

```diff
@@ -40,13 +40,27 @@ func (istio *Istio) installAddon(namespace string, del bool, service string, pat
 		}
 	}
 
-	jsonContents, err := readFileSource(patch)
-	if err != nil {
-		return st, ErrAddonFromTemplate(err)
+	jsonContents := make([]string, 0)
+	for _, patch := range patches {
+		content, err := readFileSource(patch)
+		if err != nil {
+			return st, ErrAddonFromTemplate(err)
+		}
+		jsonContents = append(jsonContents, content)
 	}
 
 	if !del {
-		_, err = istio.KubeClient.CoreV1().Services("istio-system").Patch(context.TODO(), service, types.MergePatchType, []byte(jsonContents), metav1.PatchOptions{})
+		_, err := istio.KubeClient.CoreV1().Services("istio-system").Patch(context.TODO(), service, types.MergePatchType, []byte(jsonContents[0]), metav1.PatchOptions{})
```

Comment:
Can we replace these hard-coded namespace with `namespace`? I think it would be just a little less hard coding :sweat_smile:

Reply:
Uhmm we can move it to a var at max, no room anywhere else.